reference/supportability.md

```diff
@@ -0,0 +1,159 @@
+{% endhint %}
+
+
+The archive files generated by the dump command are stored in the specified output directories. The tables below specify the path and the content that will be stored in each archive file.
+
+
+<details>
+<summary>Topology Information</summary>
+| Folder Path | Mayastor Resource | Node Name |  File Name | Description |
+| :--- | :--- | :--- | :--- | :--- |
+| ./topology/node | node | node-01 | node-01-topology.json | Topology of node-01(All node topologies will available here) |
+| ./topology/pool | pool | pool-01 | pool-01-topology.json | Topology of pool-01 (All pool topologies will available here) |
+| ./topology/volume | volume | volume-01 | volume-01-topology.json | Topology information of volume-01 (All volume topologies will available here) |
+</details>
+
+<details>
+<summary>Historical Logs</summary>
+| Folder Path | Host Name(optional) | Component | File Name |
+| :--- | :--- | :--- | :--- |
+| ./logs/core-agents | - | agent-core | loki-agent-core.log |
+| ./logs/rest | - | api-rest | loki-api-rest.log |
+| ./logs/csi-controller | - | csi-attacher | loki-csi-attacher.log |
+| ./logs/csi-controller | - | csi-controller | loki-csi-controller.log |
+| ./logs/csi-controller | - | 	csi-provisioner | loki-csi-provisioner.log |
+| ./logs/diskpool-operator | - | operator-diskpool | loki-operator-disk-pool.log |
+| ./logs/mayastor| 	node-02 | csi-driver-registrar | node-02-loki-csi-driver-registrar.log |
+| ./logs/mayastor | node-01 | csi-node | node-01-loki-csi-node.log |
+| ./logs/mayastor | node-01	 | io-engine | node-01-loki-mayastor.log |
+| ./logs/blot | node-02 | io-engine | node-02-loki-mayastor.log |
+| ./logs/etcd | node-03 | etcd | node-03-loki-etcd.log |
+</details> 
+
+<details>
+<summary>Configuration details of Kubernetes resources</summary>
+| Folder Path | Component | File Name |
+| :--- | :--- | :--- |
+| ./k8s_resources/configurations/ | agent-core (Deployment) | mayastor-agent-core.yaml |
+| ./k8s_resources/configurations/ | api-rest | mayastor-api-rest.yaml |
+| ./k8s_resources/configurations/ | csi-controller (Deployment) | mayastor-csi-controller.yaml |
+| ./k8s_resources/configurations/ | csi-node(Daemonset) |  mayastor-csi-node.yaml |
+| ./k8s_resources/configurations/ | etcd (Statefullset) | mayastor-etcd.yaml |
+| ./k8s_resources/configurations/ | loki (Statefullset) | mayastor-loki-yaml |
+| ./k8s_resources/configurations/ | operator-diskpool | mayastor-operator-disk-pool.yaml |
+| ./k8s_resources/configurations/ | promtail(Daemonset) | mayastor-promtail.yaml |
+| ./k8s_resources/configurations/ | io-engine (Daemonset) | io-engine.yaml |
+| ./k8s_resources/configurations/ | disk_pools | k8s_diskPools.yaml |
+| ./k8s_resources/configurations/ | events | k8s_events.yaml |
+| ./k8s_resources/configurations/ | all pods(deployed under the same namespace as Mayastor) | pods.yaml |
+</details>
+
+<details>
+<summary>etcd dump information</summary>
```

Comment:
Above, each word in summary starts with an upper case.

Reply:
The reason why I wrote 'etcd' and not 'ETCD' is that the Kubernetes documentation uses this convention.  [Refrence](https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/).